acquisitions/src/main/resources/thunderjet/mod-orders/features/unopen-order-with-different-fund.feature

```diff
@@ -0,0 +1,256 @@
+Feature: Unopen order and change fund distribution
+  Tickets: MODORDERS-626, MODORDERS-894, MODORDERS-1222
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json'  }
+
+    * callonce variables
+
+    @Positive
+    Scenario: Unopen order and add fund distribution with another expense class
+      # 0. Prepare finance data [create budget with two expense classes (PRN and Elec)]
```

Comment:
A nitpick, but counting from 1 is better

Reply:
fixed